### PR TITLE
Shiro's Delight (drink)

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Adds in the whacky drink that is Shiro's Delight!

Recipe:
Desoxyephedrine: 2 (drugs)
Ethylredoxrazine: 2 
Doctor's Delight: 1 
Slime: 4 (yummy)
Apple Juice: 1 (abbles)
Makes:
ShirosDelight: 1

Effects:
- Metabolizes about 25% slower
- Satiates hunger
- Satiates thirst
- Gives high amounts of ethanol when over 40u
- Give high amounts of poison when over 40u
- Very minor burn, brute and airloss healing properties
- Go fast
- Stutter
- Narcolepsy resetting above 15u
- Decent knockdown and stun removal

# TODO
- [x] drinked the dronk

# Media
delight
descript

# Changelog
:cl: SomeCabbage
- add: Added Shiro's Delight, a fun drink that is time consuming and somewhat difficult to make, but provides funny buffs. Ask Shiro for the recipe

### CODE REVIEW

File: Resources/Locale/en-US/flavors/flavor-profiles.ftl [hunk 1]
```diff
@@ -199,3 +199,6 @@ flavor-complex-sapopicante = like tomatoes and spices
 flavor-complex-graveyard = like a strong cold brew
 flavor-complex-bubbletea = like creamy sweetness
 flavor-complex-corncob = like a sick joke
+
+## parkstation
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/flavors/flavor-profiles.ftl]:
```suggestion
## Parkstation
```

File: Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/flasks.yml [hunk 2]
```diff
@@ -56,3 +56,20 @@
...
+    - type: Drink
+    - type: Sprite
+      sprite: SimpleStation14/Objects/Consumable/Drinks/shirosdelightglass.rsi
+    - type: SolutionContainerManager
+      solutions:
+        drink:
+          maxVol: 30
+          reagents:
+            - ReagentId: ShirosDelight
+              Quantity: 30
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/flasks.yml]: should this be in the flasks.yml file?

File: Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml [hunk 3]
```diff
@@ -19,3 +19,19 @@
       amount: 1
   products:
     LiquidMiasma: 1
+
+- type: reaction
+  id: ShirosDelight
+  reactants:
+    Desoxyephedrine: #meth
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml]:
```suggestion
    Desoxyephedrine: # Meth
```
nitpicky :)

File: Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml [hunk 4]
```diff
@@ -19,3 +19,19 @@
...
+      amount: 5
+    Ethylredoxrazine: #negating the alcohol effects of the delight
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml]:
```suggestion
    Ethylredoxrazine: # Negating the alcohol effects of the delight
```

File: Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml [hunk 5]
```diff
@@ -19,3 +19,19 @@
...
+      amount: 5
+    DoctorsDelight: #delight and negating the poison of the meth
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml]:
```suggestion
    DoctorsDelight: # Delight and negating the poison of the meth
```

File: Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml [hunk 6]
```diff
@@ -19,3 +19,19 @@
...
+      amount: 5
+    Slime: #squish tasty
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml]:
```suggestion
    Slime: # Squish tasty
```

File: Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml [hunk 7]
```diff
@@ -19,3 +19,19 @@
...
+      amount: 10
+    JuiceApple: #add just that little bit of extra work
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Recipes/Reactions/chemicals.yml]:
```suggestion
    JuiceApple: # Add just that little bit of extra work
```

File: Resources/Textures/SimpleStation14/Objects/Consumable/Drinks/shirosdelightglass.rsi/meta.json [hunk 8]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by Cabbage#9633",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Objects/Consumable/Drinks/shirosdelightglass.rsi/meta.json]:
```suggestion
    "copyright": "Made by DSC@Cabbage#9633 (561159087765848084)",
```

File: Resources/Prototypes/SimpleStation14/Reagents/drinks.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Reagents/drinks.yml:1]: Overdoses are pretty high

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
hehe draft review

File: Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/flasks.yml (hunk 2 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/flasks.yml]: changing it now

File: Resources/Prototypes/SimpleStation14/Reagents/drinks.yml
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Reagents/drinks.yml:1]: changed them to be actually attainable by normal drinking

**Review by Pspritechologist — APPROVED** [review]

File: Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/drinks.yml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+- type: entity
+  parent: DrinkBaseCup
+  id: DrinkShirosDelight
+  name: Shiro's Delight
+  description: A swirling solution with the slimy dissociation of an extraordinary paramedic. Due to it's thick nature it lingers in the body a bit longer than usual. Overdrinking is incredibly dangerous.
```
Comment (ashkitten) [inline Resources/Prototypes/SimpleStation14/Entities/Consumable/Drinks/drinks.yml]:
```suggestion
  description: A swirling solution with the slimy dissociation of an extraordinary paramedic. Due to its thick nature it lingers in the body a bit longer than usual. Overdrinking is incredibly dangerous.
```

File: Resources/Prototypes/SimpleStation14/Reagents/drinks.yml [hunk 10]
```diff
@@ -0,0 +1,66 @@
+- type: reagent
+  id: ShirosDelight
+  name: Shiro's Delight
+  parent: BaseDrink
+  desc: It swirls with the slime and dissociation of an extraordinary paramedic. Due to it's thick nature it lingers in the body a bit longer than usual. Overdrinking is incredibly dangerous.
```
Comment (ashkitten) [inline Resources/Prototypes/SimpleStation14/Reagents/drinks.yml]:
```suggestion
  desc: It swirls with the slime and dissociation of an extraordinary paramedic. Due to its thick nature it lingers in the body a bit longer than usual. Overdrinking is incredibly dangerous.
```

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**